mkosi/config.py

```diff
@@ -874,8 +877,12 @@ def config_default_distribution(namespace: argparse.Namespace) -> Distribution:
 
 
 def config_default_release(namespace: argparse.Namespace) -> str:
+    if (d := os.getenv("MKOSI_HOST_DISTRIBUTION")) and (r := os.getenv("MKOSI_HOST_RELEASE")):
+        hd, hr = Distribution(d), r
+    else:
+        hd, hr = detect_distribution()
```

Comment:
This is probably most easily squashed by
```suggestion
    hd: Optional[Distribution]
    hr: Optional[str]
    if (d := os.getenv("MKOSI_HOST_DISTRIBUTION")) and (r := os.getenv("MKOSI_HOST_RELEASE")):
        hd, hr = Distribution(d), r
    else:
        hd, hr = detect_distribution()
```